modules/oe_content_publication/oe_content_publication.info.yml

```
@@ -5,25 +5,41 @@ package: OpenEuropa
 type: module
 core: 8.x
 dependencies:
+  - composite_reference:composite_reference
+  - drupal:path
+  - entity_reference_revisions:entity_reference_revisions
+  - field_group:field_group
+  - inline_entity_form:inline_entity_form
+  - link:link
```

This is a core module:

```suggestion
  - drupal:link
```

---

This is fixed.